### PR TITLE
Update common and demo imperativeJobs

### PR DESCRIPTION
**Pull request by mbaldessari**
- Add initial support for imperative jobs
- Update common and demo imperativeJobs

### CODE REVIEW

**Comment by mbaldessari** [comment]
Do not merge this yet

File: common/clustergroup/values.yaml [hunk 1]
```diff
@@ -6,9 +6,29 @@ global:
...
+imperativeImagePullPolicy: Always
+imperativeActiveDeadlineSeconds: 3600
+imperativeSchedule: "*/10 * * * *"
+imperativeVerbosity: ""
+imperativeServiceAccountCreate: true
+imperativeServiceAccountName: imperative-sa
+imperativeClusterRoleName: imperative-cluster-role
+imperativeClusterRoleYaml: ""
+imperativeRoleName: imperative-role
+imperativeRoleYaml: ""
```
Comment (beekhof) [inline common/clustergroup/values.yaml:26]: I think you've gotten the wrong idea from that issue. It's only lists that have that problem. Let's discuss today

Comment (mbaldessari) [reply common/clustergroup/values.yaml:26]: No you're right. I initially had:
```
imperative:
  jobs:
  - name: n1
    ...
  - name: n2
    ...

  Image: image123
```

And that messed things up. I'll leave imperativeJobs under clusterGroup: and then group the main options under imperative:

Comment (mbaldessari) [reply common/clustergroup/values.yaml:26]: Actually let me rephrase that: if I move the "imperative:" dictionary  under clusterGroup then things do not work properly, whereas if I have a toplevel "imperative:" dictionary then things are fine.

**Comment by mbaldessari** [comment]
Ok now this branch is ready for review. This one only has the update of common from https://github.com/hybrid-cloud-patterns/common/pull/88 and it just adds a small demo ansible job.
